staking/app/StakeConnection.ts

```diff
@@ -708,6 +734,10 @@ export class StakeConnection {
       );
     }
 
+    if (!stakeAccount || !this.isLlcMember(stakeAccount)) {
```

Comment:
seems like there's a block of code above this that's duplicated a few times. consider extracting out a helper method.

Reply:
Future PR